hotel-management-app-v1/src/main/java/com/fb/hotelmanagementappv1/models/Room.java

```diff
@@ -0,0 +1,56 @@
+package com.fb.hotelmanagementappv1.models;
+
+public class Room {
+    private int id;
+    private boolean isOccupied;
+    private int roomSize;
+    private int numberOfBeds;
+    private boolean hasBalcony;
+    private boolean hasTV;
+    private int hotelId;
+
+    //Constructor
+    public Room(int id, boolean isOccupied, int roomSize, int numberOfBeds, boolean hasBalcony, boolean hasTV, int hotelId){
```

Comment:
nit:  For now this is fine, But usually, it is not recommended to have a constructor or methods with a long list of parameters (max 4 or 5). I think this is the right opportunity for you to read a little about builder design pettern. Here are some resources for you to read about it 

https://refactoring.guru/design-patterns/builder
https://www.geeksforgeeks.org/builder-design-pattern/